go/common/crypto/signature/signers/plugin/plugin.go

```diff
@@ -0,0 +1,56 @@
+// Package plugin implements the Go plugin signature signer.
+package plugin
+
+import (
+	"fmt"
+	"plugin"
+
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+)
+
+const (
+	// EntryPoint is the entry point (FactoryCtor) expected to be defined
+	// in each signer plugin.
+	EntryPoint = "GetPluginCtor"
+
+	// SignerName is the name used to identify the plugin signer.
+	SignerName = "plugin"
+)
+
+type FactoryCtor func() signature.SignerFactoryCtor
```

Comment:
Where is this used, is it there just for documentation purposes to show the type of the entry point? Maybe add a documentation comment to explain that.

Reply:
It was supposed to be actually used, but using it causes the plugin loader to error out because the runtime is too stupid to figure out that yes, the symbol does indeed match the function signature.  Added some comments.